Ignition.Sc/Ignition.Sc.csproj

```diff
@@ -295,6 +295,7 @@
     <Content Include="Components\Metadata\Taxonomy.cshtml" />
     <Content Include="Components\Metadata\PageEdit.cshtml" />
     <None Include="Properties\PublishProfiles\Ignition.Sc.pubxml" />
+    <None Include="Properties\PublishProfiles\Local.pubxml" />
```

Comment:
Instead of adding a publish profile called _Local_, roll this change back and add a publish profile called _Ignition.Sc_ to the _Ignition.Sc_ project. This will keep the project clean of tons of dead publish profiles. Publish profiles are in the Git ignore file, so you don't have to worry about your publish settings being overwritten.